app/aws-lsp-codewhisperer-runtimes/package.json

```diff
@@ -6,9 +6,7 @@
     "scripts": {
         "clean": "rm -rf out/ bin/ tsconfig.tsbuildinfo",
         "compile": "tsc --build",
-        "copy:wasm": "copyfiles -f --error ../../node_modules/@aws/fully-qualified-names/node/aws_fully_qualified_names_bg.wasm ../../node_modules/web-tree-sitter/tree-sitter.wasm out",
```

Comment:
Please make sure to note this to the team for our next release - we need to update our distribution build configuration  we use in pipeline when we release this change